lib/lexisnexis/proofer.rb

```diff
@@ -2,6 +2,7 @@ module LexisNexis
   class Proofer < Proofer::Base
     def proof_applicant(applicant, result)
       response = send_verification_request(applicant)
+      result.transaction_id = response.conversation_id
```

Comment:
what type of object is `result`? does it already have this method?

Reply:
It's a `ProoferResult`.  I had to add the `transaction_id` to it.

Reply:
`Proofer::Result` is in here: https://github.com/18F/identity-proofer-gem/pull/48